debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -87,19 +87,17 @@
                                                            "'<foo>bar</foo><foo>bar</foo>'::xml, 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11'::UUID)";
     protected static final String INSERT_STRING_TYPES_STMT = "INSERT INTO string_table (vc, vcv, ch, c, t, b) " +
                                                              "VALUES ('aa', 'bb', 'cdef', 'abc', 'some text', E'\\\\000\\\\001\\\\002'::bytea)";
-    protected static final String INSERT_NUMERIC_TYPES_STMT = "INSERT INTO numeric_table (si, i, bi, r, db, ss, bs, b) " +
-                                                              "VALUES (1, 123456, 1234567890123, 3.3, 4.44, 1, 123, true)";
-    protected static final String INSERT_FP_TYPES_NO_DECIMAL_STMT = "INSERT INTO numeric_table (si, i, bi, r, db, ss, bs, b) " +
-                                                                    "VALUES (1, 123456, 1234567890123, 3, 4, 1, 123, true)";
-    protected static final String INSERT_NUMERIC_DECIMAL_TYPES_STMT = "INSERT INTO numeric_decimal_table (d, dzs, dvs, n, nzs, nvs) " +
-                                                                      "VALUES (1.1, 10.11, 10.1111, 22.22, 22.2, 22.2222)";
-    protected static final String INSERT_NUMERIC_DECIMAL_TYPES_NO_DECIMAL_STMT = "INSERT INTO numeric_decimal_table (d, dzs, dvs, n, nzs, nvs) " +
-                                                                                 "VALUES (1, 10, 10, 22, 22, 22)";
+    protected static final String INSERT_NUMERIC_TYPES_STMT =
+            "INSERT INTO numeric_table (si, i, bi, r, db, r_nodec, db_nodec, r_nan, db_nan, r_inf, db_inf, ss, bs, b) " +
```

Comment:
What is "nodec"? Maybe we can find a more descriptive name?

Reply:
nodec means no decimals